include/zephyr/usb/usbd.h

```diff
@@ -62,7 +62,7 @@ extern "C" {
  */
 struct usbd_desc_node {
 	/** slist node struct */
-	sys_snode_t node;
+	sys_dnode_t node;
```

Comment:
Where are you using the features that dlist has that are not found in slist? The commit message mentions "next commit" but it is not in next commit and even in the last commit I didn't find (maybe I just missed) where the dlist is really needed over slist.